docs/src/dev/provider/index.asciidoc

```diff
@@ -971,19 +1133,16 @@ extensible nature of Gremlin Server, it is difficult to provide an authoritative
 It is however possible to describe the core communication protocol using the standard out-of-the-box configuration
 which should provide enough information to develop a driver for a specific language.
 
-image::gremlin-server-flow.png[width=300,float=right]
-
 Gremlin Server is distributed with a configuration that utilizes link:http://en.wikipedia.org/wiki/WebSocket[WebSocket]
 with a custom sub-protocol.  Under this configuration, Gremlin Server accepts requests containing a Gremlin script,
 evaluates that script and then streams back the results.  The notion of "streaming" is depicted in the diagram to the
 right.
 
-The diagram shows an incoming request to process the Gremlin script of `g.V()`.  Gremlin Server evaluates that script,
-getting an `Iterator` of vertices as a result, and steps through each `Vertex` within it.  The vertices are batched
-together given the `resultIterationBatchSize` configuration.  In this case, that value must be `2` given that each
-"response" contains two vertices.  Each response is serialized given the requested serializer type (JSON is likely
-best for non-JVM languages) and written back to the requesting client immediately.  Gremlin Server does not wait for
-the entire result to be iterated, before sending back a response.  It will send the responses as they are realized.
+Let's use the incoming request to process the Gremlin script of `g.V()` as an example.  Gremlin Server evaluates that 
+script, getting an `Iterator` of vertices as a result, and steps through each `Vertex` within it.  The vertices are
+batched together into an HTTP chunk.  Each response is serialized given the requested serializer type (GraphBinary is
+recommended) and written back to the requesting client immediately.  Gremlin Server does not wait for the entire result
```

Comment:
the docs keep mentioning "GraphBinary is recommended" but do we say anywhere why this is the case? ultimately, GraphBinary will effectively be the only option for drivers, so perhaps the docs should just be more clear that this is just a M1 recommendation?

Reply:
Added new "serializer" section to explain more in-depth why GraphBinary is recommended.